README.md

```diff
@@ -833,6 +834,8 @@ If you want to display the idle state, you can add it to [app.component.html](ht
 
 ## Variables
 
+Events that you can subscribe to deal with login and logout (signOut) events
+
 ```typescript
 // Events that you can subscribe to
 public onLogin  : EventEmitter<null>;
```

Comment:
Please update the two events with the naming convention (`onSignIn`, `onSignOut`).
It must be changed everywhere : sample, service & doc.
```suggestion
public onSignIn  : EventEmitter<null>;
```